keyboards/keebio/iris/keymaps/blucky/keymap.c

```diff
@@ -0,0 +1,88 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _QWERTY 0
```

Comment:
If you replace Lines 5-8 with the following, you don't have to manually define the indexes for every layer.

```c
enum layer_names {
    _QWERTY,
    _LAYER1,
    _LAYER2,
    _LAYER3
};
```

Reply:
changed